shared/helpers/view.js

```diff
@@ -14,6 +14,16 @@ module.exports = function (Handlebars) {
     viewOptions = options.hash || {};
     app = getProperty('_app', this, options);
 
+    _.forEach(_.omit(viewOptions, 'templateName'), function(value, key) {
+      if (
+        key.match(/[A-Z]/g) &&
+        !app.modelUtils.isModel(value) &&
+        !app.modelUtils.isCollection(value)
+      ) {
+        throw new Error('Template properties should be passed in snake_case: "' + key + '"')
```

Comment:
i'm suggesting it's not reaaalllly a bug with rendr-handlebars, it's more HTMLs fault?

We should probably check against it like you're suggesting, but just letting know you why it's happening and a possible work around.